test/complex-benchmark.js

```diff
@@ -17,36 +18,42 @@ function formatNumber(number) {
 }
 
 let suite = new benchmark.Suite()
+let out // eslint-disable-line no-unused-vars
```

Comment:
Can we remove it if it is not used?

Reply:
Do we need to add `out =` to benchmarks?

Reply:
JIT can throw out content of pure functions if result isn't used. It's relevant for simple benchmark but not complex. I will remove it.